exchanges/stream/websocket_types.go

```diff
@@ -93,9 +92,14 @@ type Websocket struct {
 	features          *protocol.Features
 
 	// Standard stream connection
-	Conn Connection
+	Conn            Connection
+	UnAuthHandler   func([]byte) error
+	UnAuthBootstrap func(Connection) error
+
 	// Authenticated stream connection
-	AuthConn Connection
+	AuthConn      Connection
```

Comment:
Down the line, I would love for AuthConn to be removed. An auth connection is just another connection. At present, any connection issues for unauth/auth affect the other and it doesn't make much sense for the direction we're heading with multiple websocket connection types